submissions/SamVal007/task-JS-dom/style.css

```diff
@@ -0,0 +1,236 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  margin: 0;
+  height: 100vh;
+  font-family: Snell Roundhand, cursive;
+  background-color: #f5f5f5;
+}
+
+ul {
```

Comment:
Styling by tag is risky: this will style ALL ul-s in a page. Not quite applicable for this small task, but better to keep in mind.

Reply:
Also agree, usual dont do it like this, but there is only one UL, so that's why.
But ok, let's do it better!
Very Big Thanks for helping!!!
And Happy New Year!:)